pkg/koordlet/resmanager/cpu_burst.go

```diff
@@ -167,7 +166,6 @@ func NewCPUBurst(resmanager *resmanager) *CPUBurst {
 	return &CPUBurst{
 		resmanager:       resmanager,
 		executor:         executor,
-		cgroupReader:     resourceexecutor.NewCgroupReader(),
```

Comment:
just init plugin (e.g. CPUBurst.cgroupReader) with resmanager.cgroupReader, since this will be mv to qos-manager

Reply:
done